config/defaults.conf

```diff
@@ -326,13 +326,13 @@ pair_attn = 1  // If true, use attn in sentence-pair classification/regression t
 d_hid_attn = 512  // Post-attention LSTM state size.
 shared_pair_attn = 0  // If true, share pair_attn parameters across all tasks that use it.
 d_proj = 512  // Size of task-specific linear projection applied before before pooling.
-classifier_loss_fn = ""  // Classifier loss function. Used only in some tasks (notably
-                         // span-related tasks),  not mlp/fancy_mlp. Currently supports
+classifier_loss_fn = "softmax"  // Classifier loss function. Used only in some tasks (notably
```

Comment:
Maybe for a future PR, but if this is specific to the span classifier, give it a more specific name like `span_classifier_loss` ?

Or just use the same parameter that's used for the rest of tasks, assuming the functionality is more or less the same.